core/src/main/scala/io/qbeast/core/transform/LinearTransformation.scala

```diff
@@ -256,27 +243,19 @@ object LinearTransformationUtils {
 
   /**
    * Creates a LinearTransformationUtils object that contains useful functions that can be used
-   * outside of the LinearTransformation class.
-   * @param minNumber
-   * @param maxNumber
-   * @param orderedDataType
-   * @param seed
-   * @return
+   * outside the LinearTransformation class.
    */
-
   def generateRandomNumber(min: Any, max: Any, seed: Option[Long]): Any = {
     val r = if (seed.isDefined) new Random(seed.get) else new Random()
     val random = r.nextDouble()
-
     (min, max) match {
       case (min: Double, max: Double) => min + (random * (max - min))
       case (min: Long, max: Long) => min + (random * (max - min)).toLong
       case (min: Int, max: Int) => min + (random * (max - min)).toInt
       case (min: Float, max: Float) => min + (random * (max - min)).toFloat
-      case (min, max) =>
+      case (min, _) =>
```

Comment:
maybe put `case(_, _)` and use the min/max values outside the match to output the error